benchmark/llama.py

```diff
@@ -82,7 +40,7 @@ def collect_metrics(benchmark_id, continue_metric_collection):
     conn.close()
 
 
-def run_benchmark(branch: str, commit_id: str, commit_msg: str, num_tokens_to_generate=100):
+def run_benchmark(logger: Logger, branch: str, commit_id: str, commit_msg: str, num_tokens_to_generate=100):
```

Comment:
is there a way to only pass the logger and a class that wrapps around your `continue_metric_collection` . 
This way the only thing people need when adding a new file with run_benchmark is: 
1. Call start() on the class
2. compute some stuff
3. call .record() (for example) which take the dict of inputs, and adds them to the data, commit and close! 

completely up to debate, try to find the simplest way to do it! 🤗

Reply:
Yes that's probably the best path forward! Will iterate and let you know when I have something nicer

Reply:
perfect thanks